From private repo

v-alje opened the pull request.

yijichen commented:
#sign-off

azurecla commented:
Hi __@yijichen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (yijichen). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rolyon committed:
Removed (Preview) roles, included non-Microsoft provider operations

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-updateroles-321225

v-nagta committed:
updated next steps and msiam access topic

mmacy committed:
[ACR] regional failure clarification

mmacy committed:
[ACR] geo-redundancy vs. geo-replication

mmacy committed:
[ACR] acro fixes

v-nagta committed:
Adding Custom role to TOC

v-nagta committed:
adding custom role doc to the develop folder

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
[ACI] freshness pass (batch 02 - tutorials)

mmacy committed:
[ACI] tutorial 1 freshness + include

mmacy committed:
[ACI] tutorial 2 freshness

mmacy committed:
Merge branch 'master' of MicrosoftDocs/azure-docs-pr into aci-freshness-201803-pass2

mmacy committed:
[ACI] tutorial ownership + cgroup image

mmacy committed:
[ACI] remove cursor from cgroup diagram

msvijayn committed:
OMS Migration docs & revised docs for Log Alerts in Azure Alerts

msvijayn committed:
Merge branch 'release-monitor-gartner' of https://github.com/MicrosoftDocs/azure-docs-pr into gartner-vijayn

msvijayn committed:
Correcting link error

msvijayn committed:
Correcting Link Issue

msvijayn committed:
Fixing Link mistakes

msvijayn committed:
typo cleanup

msvijayn committed:
Typos & Grammer mistakes cleanup

msvijayn committed:
Typo & Grammer clean-up

Commit:
Added Timed Metadata spec

Commit:
updates

Commit:
edits

Commit:
edits

Commit:
edits

msvijayn committed:
Pricing legal text change

msvijayn committed:
typo mistake

msvijayn committed:
Typo cleanup

msvijayn committed:
Retry Build

apwestgarth committed:
Updates to relnotes re swap issue

rolyon committed:
Removed non-Microsoft providers

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-updateroles-321225

Commit:
updated author and date

Commit:
Edits

v-geberr committed:
0319-batch-test-tutorial-with-entities

mmacy committed:
[ACI] tutorial include: use Important box

Commit:
fixed typo

Commit:
added author

apwestgarth committed:
Updated release notes to cover issues with
certificate validation, retrieving root certificate and swap behaviour

apwestgarth committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

apwestgarth committed:
fixed date string and spelling

KumudD committed:
feedback incorp

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into qscreateloadbalancercli

KumudD committed:
updated art

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into qscreateloadbalancercli

juhacket committed:
Fix sample service names

bwren committed:
Computer groups and Power BI

v-geberr committed:
tokenization based on culture affects data extraction

apwestgarth committed:
Edits to improve quality

apwestgarth committed:
Updated phrasing to remove we from sentence

Commit:
automation: metadata cleanup #5

v-geberr committed:
fix for none intent low score confusion

Commit:
updates

Commit:
acrolinx updates

Commit:
acrolinx feedback

Commit:
acrolinx feedback

mmacy committed:
[ACI] tut3: more refresh work

v-geberr committed:
touch to build

eringreenlee committed:
fixed alert 105 strings

apwestgarth committed:
removed date stamps

v-geberr committed:
entities back in

curtand committed:
[Azure AD windows Server] add mention of adding static IP address to VM in portal

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#5974

bwren committed:
agent health, ap insights, alert management

mmacy committed:
[ACI] more org + next step for tut3

apwestgarth committed:
fix to modified date

bwren committed:
remove log search api

davidmu1 committed:
ownership change and meta cleanup

bwren committed:
table

msvijayn committed:
Review comment changes

msvijayn committed:
Review comment changes

msvijayn committed:
Updating as per review comments

msvijayn committed:
Added pricing link

v-geberr committed:
wip

v-geberr committed:
changed  date

msvijayn committed:
Title change

hope for build commit

v-nagta committed:
USER STORY 1213229 SaaS App Tutorial: Domo Tutorial Update

v-nagta committed:
USER STORY 1213233 SaaS App Tutorial: Amazon Web Services (AWS) Update

v-nagta committed:
en-us removed from the URL blocking issue updated

v-nagta committed:
updated

MSSedusch committed:
SAP: add pacemaker article

MSSedusch committed:
SAP: add pacemaker guide to toc

MSSedusch committed:
remove broken links

MSSedusch committed:
SAP: improve intro

MSSedusch committed:
SAP: improve SP steps

bwren committed:
table

MSSedusch committed:
SAP: add overview

Commit:
Script samples

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into samples

Commit:
Samples.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into samples

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into samples

MGoedtel committed:
updated article with additional required step

mmacy committed:
[ACI] multi-container wording

barclayn committed:
adding the global admin article to staging

MGoedtel committed:
updated to align with taxonomy and LA guidance

v-geberr committed:
rss search for updated docs referencing 'LUIS'

neilpeterson committed:
[AKS] fixes MicrosoftDocs/azure-docs#6024

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into static0319

mmacy committed:
[ACI] minor grammar fixes

curtand committed:
add link to static IP article

mmacy committed:
[ACI] landing page update

v-geberr committed:
edits

mmacy committed:
[ACR] remove backup section

Commit:
ASDK TOC updates and redirects

mmacy committed:
[ACI] one line break

Commit:
added note markdown

billmath committed:
updating per sree

billmath committed:
updating per issues

cynthn committed:
Update virtual-machines-common-sizes-general.md

msvijayn committed:
As per review comments

barclayn committed:
adding the article to the TOC and changing the h1

msvijayn committed:
Updated as per review comments

Commit:
Merge pull request #35787 from billmath/issues

updating per issues

Commit:
Merge pull request #35786 from billmath/sree

updating per sree

Commit:
fixed markdown error

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#5919

Updating SLA link

mmacy committed:
[ACI] no line break

barclayn committed:
fixing Tom's id

Ja-Dunn committed:
Update log-analytics-powerbi.md

date fix

rmca14 committed:
Merge pull request #35683 from apwestgarth/master

Updated Release Notes

msvijayn committed:
updated as per review comments

vhorne committed:
fix redirect problem per Tyson

mmacy committed:
[ACI] para tag

carolinacmoravia committed:
C14104: Missing asteriks for bold type

Hello, @sharonlo101, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

Commit:
Task 1035: Split POP tables into separate topics

PRMerger14 committed:
Merge pull request #35792 from iainfoulds/patch-19

[VM] Fixes MicrosoftDocs/azure-docs#5919

Commit:
minor rearrangements

Ja-Dunn committed:
Merge pull request #35685 from v-geberr/0319-entity-extraction-tokenization-by-culture

tokenization based on culture affects data extraction

eringreenlee committed:
fixed wording 105

douglaslMS committed:
Removed broken link to bookmark in redirected topic.

Ja-Dunn committed:
Merge pull request #35682 from bwren/kusto-ga-2

Kusto GA - Computer groups and Power BI

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0319-batch-test-tutorial-with-entities

lindavr committed:
fix metadata issue

v-shils committed:
Merge pull request #34125 from mmacy/acr-image-storage

[ACR] image storage

rjagiewich committed:
Merge pull request #34083 from v-nagta/customrole

USER STORY 1174649 SaaS App Tutorial: Custom Role Setup

mmacy committed:
[ACI] metadata to include file

rmca14 committed:
Merge pull request #34075 from KumudD/qscreateloadbalancercli

QuickStart : Load balance VMs - Azure CLI

ktoliver committed:
Merge pull request #35781 from jeffgilb/asdk-update2

ASDK TOC updates and redirects

Ja-Dunn committed:
Merge pull request #35699 from georgewallace/autometadata

automation: metadata cleanup #5

PRMerger18 committed:
Merge pull request #32601 from cloudhandler/patch-1

Update dynamic group doc example rules

PRMerger18 committed:
Merge pull request #35799 from douglaslMS/adf-fix-bookmark

Removed broken link to bookmark in redirected topic.

mmacy committed:
[ACI] update CLI icon

apwestgarth committed:
Update azure-stack-app-service-release-notes-update-one.md

Clarification around new SAN name requirement

rjagiewich committed:
Merge pull request #35726 from bwren/kusto-ga-3

Kusto GA - agent health, ap insights, alert management

apwestgarth committed:
Update azure-stack-app-service-release-notes-update-one.md

ktoliver committed:
Merge pull request #35613 from msvijayn/gartner-vijayn

Docs for Mar 21 release

lindavr committed:
another fix

Lisaco88 committed:
Merge pull request #35718 from curtand/static0319

[Azure AD windows Server] adding static IP address to VM in portal

Ja-Dunn committed:
Merge pull request #35096 from srrengar/sravworkbranch

Updated WAD configuration section

ktoliver committed:
add spaces so bulleted list renders correctly

add spaces so bulleted list renders correctly

Commit:
Merge pull request #32606 from cloudhandler/patch-2

Update CA best practices- clarity fix

tynevi committed:
Merge pull request #35796 from vhorne/fix-redirect

fix redirect problem per Tyson

Commit:
Minor feedback updates

Brenduns committed:
Release note update for 1802

Commit:
Merge pull request #35795 from carolinacmoravia/patch-652

C14104: Missing asteriks for bold type

mmacy committed:
[ACI] logo fix

Commit:
Merge pull request #35723 from tfitzmac/0319ad

clarified step to assign app to role

Commit:
Merge pull request #35800 from lindavr/fixwpmeta

fix metadata issue

v-shils committed:
Merge pull request #35566 from mmacy/aci-freshness-201803-pass2

[ACI] freshness pass (batch 02 - tutorials)

apwestgarth committed:
Update azure-stack-app-service-release-notes-update-one.md

Update date of edit

ktoliver committed:
Merge pull request #33900 from cloudhandler/patch-5

update gotomeeting sso instructions

rjagiewich committed:
Merge pull request #35758 from MSSedusch/sap-pacemak

SAP: add pacemaker article

Commit:
Merge pull request #35804 from jeffgilb/jeff320

Minor feedback updates

christiankuhtz committed:
removed sign-up instructions

christiankuhtz committed:
tweak

rjagiewich committed:
Merge pull request #35773 from MGoedtel/UpdateSvcFbrLACfg320

updated article with additional required step

v-shils committed:
Merge pull request #35788 from cynthn/patch-68

Update virtual-machines-common-sizes-general.md

v-shils committed:
Merge pull request #35789 from juhacket/master

Fix sample service names

PRMerger12 committed:
Merge pull request #35801 from apwestgarth/patch-5

Patch 5 - Certificate Requirement Clarity for New Deployments

Commit:
Task 1035: Edits

PRMerger14 committed:
Merge pull request #35721 from eringreenlee/master

fixed alert 105 strings

christiankuhtz committed:
tweaks

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into samples

PRMerger14 committed:
Merge pull request #35805 from Brenduns/march-20

Release note update for 1802

mmacy committed:
[ACR] fixes MicrosoftDocs/azure-docs#5856

Brenduns committed:
Update asdk-release-notes.md

Format fix.

mmacy committed:
[ACR] fixes MicrosoftDocs/azure-docs#5856

rolyon committed:
Resolved conflict in Microsoft.OperationsManagement section

CarlRabeler committed:
[sql db] 4 TB per database

PRMerger18 committed:
Merge pull request #35809 from mmacy/acr-issue-5856

[ACR] fixes MicrosoftDocs/azure-docs#5856

Lisaco88 committed:
Merge pull request #35741 from davidmu1/b2creassign

ownership change and meta cleanup

barclayn committed:
making changes as requested by the reviewer

cynthn committed:
Update tutorial-manage-vm.md

v-geberr committed:
fix batch for low scoring intents

CarlRabeler committed:
Merge branch 'master' into MI4TB

v-geberr committed:
rss doesn't work without culture

CarlRabeler committed:
[sql db] open stuff

CarlRabeler committed:
more

BryanTrach-MSFT committed:
(forumsteam) fixes MicrosoftDocs/azure-docs#6006

Updated link for [Understanding cost reports] to fix typo. Link should now work.

v-shils committed:
Merge pull request #35797 from dksimpson/dean-1035

Task 1035: Split POP tables into separate topics

BryanTrach-MSFT committed:
(forumsteam) fixes MicrosoftDocs/azure-docs#6006

Updated link to  - name: Understanding cost reports  href: understanding-cost-reports.md, which fixed typo. Link should now work.

PRMerger13 committed:
Merge pull request #35810 from CarlRabeler/MI4TB

[sql db] 4 TB per database

PRMerger13 committed:
Merge pull request #35817 from CarlRabeler/open

Open

v-geberr committed:
grey border

michhar committed:
Connect model management setup to the deploy web service

* Removed the command to create the realtime web service as this is should appear in the `model-management-service-deploy.md` article (as an alternative, but alongside, commands to do this step-by-step)
* Added a link for continuation on to the `model-management-service-deploy.md` article as a next step
* Added brief summary of the article in lieu of the web service creation command
* Note:  a separate PR with modifications to the `model-management-service-deploy.md` article will be created to migrate the web service creation command there

Lisaco88 committed:
Merge pull request #35806 from christiankuhtz/patch-85

removed sign-up instructions

rjagiewich committed:
Merge pull request #35763 from jimdial/samples

Samples

rmca14 committed:
Merge pull request #34764 from rolyon/rolyon-updateroles-321225

 [Azure AD RBAC] Update built-in role definitions 321225

PatAltimore committed:
Fix typo

cynthn committed:
Update tutorial-manage-vm.md

Brenduns committed:
Update asdk-release-notes.md

Fixing typo.

GitHubber17 committed:
removed leftover asterisk syntax

v-geberr committed:
acro check

PRMerger17 committed:
Merge pull request #35818 from BryanTrach-MSFT/patch-7

(forumsteam) fixes MicrosoftDocs/azure-docs#6006

Commit:
Removed en-us

rjagiewich committed:
Merge pull request #35820 from BryanTrach-MSFT/patch-8

(forumsteam) fixes MicrosoftDocs/azure-docs#6006

Commit:
Fixed text.

PRMerger19 committed:
Merge pull request #35824 from michhar/patch-1

Connect model management setup to the deploy web service

PatAltimore committed:
Fix script typo

Brenduns committed:
Update asdk-release-notes.md

Typo2

GitHubber17 committed:
Merge pull request #35617 from johndeu/master

Added new Timed Metadata Specification

Lisaco88 committed:
Merge pull request #35505 from v-nagta/customroletoc

Adding Custom role to TOC

GitHubber17 committed:
Merge pull request #35774 from v-geberr/0320-rss-link-in-toc

rss search for updated docs referencing 'LUIS'

michhar committed:
Update model-management-service-deploy.md

Proposed updates

* Added the single service creation command (originally in `deployment-setup-configuration.md` setup article (a separate PR is open for the removal of this from that article)
* What you need to get started Section
* Added clarity to the statement about schema generation being optional 
* Added DataTypes.STANDARD as this critical type was missing, although widely used in samples

Commit:
Merge pull request #35826 from PatAltimore/brent2

Fix typo

Commit:
Merge pull request #35821 from Brenduns/march-20

Update asdk-release-notes.md

lindavr committed:
set expanded to false

GitHubber17 committed:
Merge pull request #35666 from v-geberr/0319-batch-test-tutorial-with-entities

0319-batch-test-tutorial-with-entities

v-shils committed:
Merge pull request #35777 from barclayn/security-global-admin

adding the global admin article to staging

PRMerger12 committed:
Merge pull request #35828 from jimdial/patch-99

Fixed text.

j-martens committed:
AML: Updating obsolete author information

mumian committed:
add a spark release note

j-martens committed:
AML: fix acrolinx

PRMerger17 committed:
Merge pull request #35754 from v-nagta/domo

USER STORY 1213229 SaaS App Tutorial: Domo Tutorial Update

PRMerger18 committed:
Merge pull request #35833 from mumian/0320-spark-releasenote

add a spark release note

GitHubber17 committed:
Merge pull request #35827 from jimdial/patch-98

Removed en-us

GitHubber17 committed:
Merge pull request #35830 from lindavr/collapsetoc

[Security toc] set expanded to false

ktoliver committed:
Merge pull request #35779 from mmacy/aci-landing-page

[ACI] landing page update

Ja-Dunn committed:
Merge pull request #35831 from j-martens/master

AML: Updating obsolete author information

j-martens committed:
AML: updating metadata

Commit:
Updated registration article

michhar committed:
Update model-management-service-deploy.md

Commit:
updated article

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

SumitParikh committed:
Test change of date

PRMerger16 committed:
Merge pull request #35838 from j-martens/master

AML: updating metadata

ktoliver committed:
change ms.date year and fix typo

change ms.date year and fix typo

v-alje committed:
Merge pull request #35842 from MicrosoftDocs/FromPublicRepo

From public repo

rmca14 committed:
Merge pull request #35776 from neilpeterson/aks-issue-6024

[AKS] fixes MicrosoftDocs/azure-docs#6024

rjagiewich committed:
Merge pull request #35840 from jeffgilb/320

Updated registration article

ktoliver committed:
Merge pull request #35755 from v-nagta/amazonaws

 USER STORY 1213233 SaaS App Tutorial: Amazon Web Services (AWS) Update

GitHubber17 committed:
Merge pull request #35815 from cynthn/patch-69

Update tutorial-manage-vm.md

GitHubber17 committed:
Merge pull request #35829 from michhar/patch-2

More consistent flow with model management and fixes

GitHubber17 committed:
Merge pull request #35845 from SumitParikh/master

Test change of date
